### PR TITLE
Wepack ToDo List implementation first setup

### PR DESCRIPTION
**Pull request by richie1988**
created the the todo list clone vision of the app

### CODE REVIEW

**Review by julie-ify — COMMENTED** [review]
Hi @richie1988,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

* Kindly remove the section description on the Key features section and provide a live link or remove the live demo section of your readme file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by iambenkis — COMMENTED** [review]
Hi @richie1988  , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 

- [ ] You have submitted a project with a README that does not have all the headers as it is in the README provide by microverse 😄 . Kindly add all the headers in your README and ask again for review 👍 

image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me `@iambenkis` in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -137,7 +130,6 @@ If you like this project, give it a star .
 
 I would like to thank the Microverse Team and the founder of Microverse for the opportunity is giving the developers around the world to archieve there dreams.
 
-
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (MahmoudBakr23) [inline README.md:135]: - [ ] Please fix the MIT License link for a professional README file and repo. 🙏

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @richie1988 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Well done creating the hard-coded tasks array in JS
- Good work iterating over the created array and displaying them in the DOM
- Nice design
- Descriptive PR
- Correct Linters

However, there is always room for improvement.

> I'd like to remind you that your long hours of work are appreciated. However, the code review process is for improving your code, not criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

N/A

Cheers, and Happy coding!👏👨‍💻💻

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**